File: code/products/redis/connect.py
import redis


def main():
    redis_uri = 'REDIS_URI'
    redis_client = redis.from_url(redis_uri)

    redis_client.set('key', 'hello world')
    key = redis_client.get('key').decode('utf-8')

    print('The value of key is:', key)


if __name__ == '__main__':
    main()

File: code/products/redis/connect.java
import redis.clients.jedis.JedisPooled;

public class RedisExample {

    public static void main(String[] args) {
        if (args.length != 1) {
            throw new IllegalArgumentException("Expected only one argument redis URI");
        } else {
            JedisPooled jedisPooled = new JedisPooled(args[0]);
            jedisPooled.set("key", "hello world");
            System.out.println("The value of key is: " + jedisPooled.get("key"));
        }
    }
}
